presto-spi/src/main/java/com/facebook/presto/spi/block/SingleMapBlock.java

```diff
@@ -124,7 +124,7 @@ public Block getLoadedBlock()
 
     int[] getHashTable()
```

Comment:
Now that we are making changes around this method, I think we should mark this method as `@Nullable`.